reference/Task%20Force%20Noctem%20Vehicles/header.sqe

```diff
@@ -1,4 +1,4 @@
 version=53;
 name="Task Force Noctem Vehicles";
 author="TomHighway";
-category="CNTO_Defaults";
+category="CNTODefaults";
```

Comment:
Might belong under "CNTO_Standard_Factions" catergory rather than defaults this way keeps all other items which are very simular i.e. same catergory together. Viscom, PvP, Coop modules and scope box with new MAT box all very simular items which are mandatory. TFN Vehicles are optional so feels wrong in this catergory.

Reply:
Yeah it's a mistake. I will fix this.